src/app/components/user-profile/user-profile.component.ts

```diff
@@ -1,21 +1,32 @@
 import { Component, OnInit } from '@angular/core';
 import { AuthStore, User, AuthQuery } from 'src/app/shared/store';
+import { UserService } from 'src/app/shared/services/user/user.service';
 
 @Component({
   selector: 'app-user-profile',
   templateUrl: './user-profile.component.html',
   styleUrls: ['./user-profile.component.scss']
 })
 export class UserProfileComponent implements OnInit {
-   user : Partial<User>
-   test : any 
-  
-  constructor(public authQuery: AuthQuery) { }
+  user : Partial<User>
+  test : any 
+  reservations: []
+  boughtTickets: []
+
+  constructor(public authQuery: AuthQuery, private userService: UserService) { }
 
   ngOnInit() {
     this.authQuery.user$.subscribe((user) => {
       this.user = user;
     })
```

Comment:
ovo treba proveriti, meni se desavalo kad ovako napisem subscribe na query, u ngOnInit, on se svaki put ponovo subscribe-uje, i ako otvoris user profile 5 puta on se vec 5 puta subscribe-ovao jer uvek poziva ngOnInit svaki put reaguje na promenu u store-u... radi sve ali se konstantno subscribe-uje i to mozda moze da pojede neku memoriju ili sta ja znam...  ja sam to resio tako sto sam u ngOnDestroy napisao unsubscribe

Reply:
U pravu si. Treba izgleda da na sve sto se u initu subscribuje da se u onDestroy unsub